tvb_build/docker/Dockerfile-run

```diff
@@ -76,4 +78,5 @@ WORKDIR $USER_HOME/tvb-root
 # bind port 8080
 # MOUNT -v [local- ~/TVB ]:$TVB_STORAGE
 
+USER root
```

Comment:
Can't we stay with user TVB for running tvb?
For the public deploys especially it will be better like that, for security reasons